src/Symfony/Component/Form/Extension/Validator/Constraints/FormValidator.php

```diff
@@ -259,7 +259,7 @@ private static function resolveValidationGroups(string|GroupSequence|array|calla
         return (array) $groups;
     }
 
-    private static function getConstraintsInGroups(array $constraints, string $group): array
+    private static function getConstraintsInGroups(array $constraints, string|array $group): array
```

Comment:
Method updated to deal with arrays.
In theory it could also be a `GroupSequence`, but the code doesn't deal with that.
Would you like to have a look?